source/manual/deploying-terraform.html.md

```diff
@@ -62,12 +65,12 @@ gds govuk terraform -e <environment> -p <project> -s <stack> -a <action> -r <aws
 Where:
 
 1. `<github_username>` is the name of your GitHub account
-1. `<github_token>` is the GitHub token that you created as described [above](#3-get-github-credentials)
-1. `<environment>` is the govuk environment you want to deploy to. E.g. `integration`,`staging`
-1. `<project>` is the terraform project that you want to deploy. E.g. `app-gatling`
-1. `<stack>` is the govuk stack you want to deploy to. E.g. `blue` (which is usually for `app-` projects), `govuk` (which is usually for `infra-` projects)
-1. `<action>` is the terraform action you want to perform. E.g. `plan`, `apply`
-1. `<aws_role>` is the govuk aws role you want to use for terraforming. E.g. `govuk-integration-admin`
+2. `<github_token>` is the GitHub token that you created as described [above](#3-get-github-credentials)
+3. `<environment>` is the govuk environment you want to deploy to. E.g. `integration`,`staging`
+4. `<project>` is the terraform project that you want to deploy. E.g. `app-gatling`, `infra-security`
+5. `<stack>` is the govuk stack you want to deploy to. E.g. `blue` (which is usually for `app-` projects), `govuk` (which is usually for `infra-` projects)
+6. `<action>` is the terraform action you want to perform. E.g. `plan`, `apply`
+7. `<aws_role>` is the govuk aws role you want to use for terraforming. E.g. `govuk-integration-admin`
```

Comment:
Leaving all the numbers as `1` is good practice. Markdown automagically sets them to a numbered list, and HTML takes care of calculating the numbers. Giving them exact numbers means there's more work to keep them correct in future.